### PR TITLE
add prometheus metrics instrumentation endpoint and active series metric

### PR DESCRIPTION
**Pull request by jtlisi**
This PR adds a prometheus metrics endpoint at `/prometheus/metrics`. It exposes the number of active series and default Golang metric provided by the prometheus client lib.

### CODE REVIEW

**Comment by Dieterbe** [comment]
Did you use proper dep workflow ? I would expect a pinning of the repo in gopkg.toml

**Comment by jtlisi** [comment]
@Dieterbe We all already importing the package due to the presence of PromQL, so no dep changes required.

File: mdata/init.go [hunk 1]
```diff
@@ -65,6 +67,12 @@ var (
 
 	schemasFile = "/etc/metrictank/storage-schemas.conf"
 	aggFile     = "/etc/metrictank/storage-aggregation.conf"
+
+	promActiveSeries = promauto.NewGauge(prometheus.GaugeOpts{
+		Namespace: "metrictank",
+		Name:      "series_active",
+		Help:      "Current # of active series",
+	})
```
[the remark this answers is not in the document]
Comment (jtlisi) [reply mdata/init.go]: I'll update it to be metrics, this is a metrictank specific metric so it should use metrictank nomenclature.

File: mdata/init.go [hunk 2]
```diff
@@ -65,6 +67,12 @@ var (
 
 	schemasFile = "/etc/metrictank/storage-schemas.conf"
 	aggFile     = "/etc/metrictank/storage-aggregation.conf"
+
+	promActiveSeries = promauto.NewGauge(prometheus.GaugeOpts{
```
Comment (Dieterbe) [inline mdata/init.go]: var should be named promActiveMetrics

Comment (jtlisi) [reply mdata/init.go]: :+1:

**Pull request merged**